app/components/forms/events/view/videoroom-form.hbs

```diff
@@ -53,7 +53,8 @@
   </div>
   <div class="field">
     <label>{{t 'Additional Information'}}</label>
-    <Textarea
+    <Widgets::Forms::RichTextEditor
```

Comment:
Need to add sanitization in server before this

Reply:
Not familiar with the server yet, should I close this PR?

Reply:
No